js/screens/LearningCurriculum/styles.js

```diff
@@ -1,40 +1,47 @@
 import { StyleSheet, Dimensions } from 'react-native';
```

Comment:
You are not using Dimensions here. Please remove the import.

Reply:
Done